ingress/icmp_linux.go

```diff
@@ -78,19 +78,19 @@ func checkInPingGroup() error {
 	if err != nil {
 		return err
 	}
-	groupID := os.Getgid()
+	groupID := uint64(os.Getegid())
```

Comment:
Can you revert this to use the old method. I don't believe it is required to use the effective group id.
```suggestion
	groupID := uint64(os.Getgid())
```

Reply:
Hi @crrodriguez is this documented? Can you share a link to it?

Reply:
there  --> https://github.com/torvalds/linux/blob/026e680b0a08a62b1d948e5a8ca78700bfac0e6e/net/ipv4/ping.c#L255
it compares the effective group id with the list